roles/rabbit-to-bd/defaults/main.yaml

```diff
@@ -0,0 +1,21 @@
+---
+# Git branches
+rabbit_to_db_git_branch: develop
+
+# Paths to services folder
+rabbit_to_db_workdir: /opt/rabbit-to-db
+
+# systemd unit folder
+rabbit_to_db_service_dir: /etc/systemd/system
```

Comment:
Just hardcode this service directory. From my perspective, this will never be changed.